app/view_utils/manage_availability_helper.rb

```diff
@@ -0,0 +1,33 @@
+module ManageAvailabilityHelper
+
+  module_function
+
+  def availability_props(community:, listing:)
+
+    {
+      i18n: {
+        locale: I18n.locale,
+        defaultLocale: I18n.default_locale,
+        localeInfo: I18nHelper.locale_info(Sharetribe::AVAILABLE_LOCALES, I18n.locale)
+      },
+      marketplace: {
+        marketplace_color1: CommonStylesHelper.marketplace_colors(community)[:marketplace_color1],
+      },
+      listing: {
+        title: listing.title,
+        image_url: path_to_listing_image(listing),
+      }
+    }
```

Comment:
It seems that camelCase (`defaultLocale`, `localeInfo`) are mixed with snake_case (`marketplace_color1` and `image_url`). Is that on purpose?